UI.PageModels/Pages/VhPage.cs

```diff
@@ -31,13 +31,11 @@ protected VhPage(IWebDriver driver, int defaultWaitTime, bool ignoreAccessibilit
         AccessibilityReportFilePath = config.AccessibilityReportFilePath;
         AccessibilityHtmlReportFilePath = config.AccessibilityHtmlReportFilePath;
         if (driver is RemoteWebDriver) Locale = "en-US";
-        ConfirmPageHasLoaded();
         CheckAccessibility();
     }
 
     private void CheckAccessibility()
     {
-        ConfirmPageHasLoaded();
         if(!AccessibilityCheck || IsLoginPage || IgnoreAccessibilityForPage) return;
```

Comment:
If you do not wait for the page to load accessibility fails because it will run the checks on a blank page. This was a common reason for failure because the hearing list had not yet loaded, and so the a11y checks failed because the page structure was still in a weird state